README.md

```diff
@@ -23,7 +23,9 @@
 [![Tweet][tweet-badge]][tweet-url]
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
```

Comment:
These empty lines were added in the precommit hook

Reply:
Yeah, this is changing the whole time 😓 I think we need to include a prettier:ignore comment to handle this properly. Sorry about weirdness this may have caused!